### ISSUE TITLE
[$20] UX/Accessibility issue - Some textboxes has double labels for screen readers

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1520]
**Describe the bug**
When I use Voice Over (screen reader) on the Basic Info form, some textboxes has double labels.

**To Reproduce**
1. Turn on 'Voice Over' 
2. Go to 'https://www.topcoder-dev.com/settings/profile'
3. Click on 'Basic Info'
4. Navigate to each textbox.

**Expected behavior**
When using the site with screen reader, each textbox should have a clear label (not duplicate).

**Screenshots**
<img width="1280" alt="screenshot 2018-09-16 21 32 19" src="https://user-images.githubusercontent.com/35376779/45600244-c6511d00-b9f9-11e8-94f0-e2e11f53f4ef.png">

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30072966 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@ThomasKranitsas can you please clarify a bit on this issue? i am confused as description doesnt seem to look proper.

**Comment by PrakashDurlabhji** [comment]
@ThomasKranitsas what is voice over? i dont see any such option on my windows 10? can you please guide thanks

**Comment by ThomasKranitsas** [comment]
I think it's called `narrator` on Windows.

**Comment by PrakashDurlabhji** [comment]
@ThomasKranitsas working on it.

**Comment by PrakashDurlabhji** [comment]
@ThomasKranitsas narrator is wokring , but on going through each checkbox , no such dialog appears in black.i guess its already solved. will i be paid something to investigate?

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30072966 has been updated - it has been assigned to gurmeetb.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
@gurmeetb can you please fix all tickets related to "screen readers"? I'll update the prize accordingly.

**Comment by ThomasKranitsas** [comment]
@gurmeetb what's the status on this one?

**Comment by gurmeetb** [comment]
@ThomasKranitsas Sorry completely missed this one. I am currently blocked in some other work, so opening it up, will revisit this later if still not fixed.

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30072966 has been updated - it has been assigned to diazz.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30072966<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```
